recipe/meta.yaml

```diff
@@ -22,6 +22,7 @@ requirements:
   build:
     - msinttypes  # [win]
```

Comment:
I don't think msinttypes is required now. Might be a good time to take this out...

Reply:
Thanks, the change looks welcome.